README.md

```diff
@@ -17,20 +18,22 @@ for information about the standalone version of cri-containerd.
 With it, you could run Kubernetes using containerd as the container runtime.
 ![cri](./docs/cri.png)
 ## Current Status
+`cri` is a native built-in plugin of containerd >= 1.1.
```

Comment:
containerd v1.1 and above.

Reply:
What do you mean by native built-in plugin? Recommend splitting up the adjectives to explain what is meant. Golang native? Built-in as in one of a set of the built-in plugins making up containerd?

Reply:
> containerd v1.1 and above.

Will do.

> What do you mean by native built-in plugin? Recommend splitting up the adjectives to explain what is meant. Golang native? Built-in as in one of a set of the built-in plugins making up containerd?

`cri` is a native plugin of containerd 1.1 and above. It is built in containerd and enabled by default.

What about this?

Reply:
Done